build.gradle

```
@@ -36,5 +36,5 @@ apply plugin: 'com.mparticle.kit'
 
 dependencies {
     testImplementation fileTree(dir: 'libs', include: ['*.jar'])
-    compileOnly 'com.onetrust.cmp:native-sdk:202308.2.0.0'
+    compileOnly 'com.onetrust.cmp:native-sdk:202308.+'
```

```suggestion
    //note that compileOnly requires kit users to define the dependency themselves
    compileOnly 'com.onetrust.cmp:native-sdk:202308.2.0.0'
```

I see that we are already using `compileOnly` - so actually we already require customers to do what you added to the readme. For example - check the dependency POM of onetrust kit on maven vs braze/appboy (or any other kit). you'll see that for the onetrust kit, we don't actually list onetrust's SDK as a `<dependency>`. for braze, you can see their sdk (and a specific version) listed.

onetrust kit POM: https://central.sonatype.com/artifact/com.mparticle/android-onetrust-kit/overview

braze kit PM: https://central.sonatype.com/artifact/com.mparticle/android-appboy-kit/overview

I think we should leave the dependency as is to keep builds deterministic, but add a comment so people know what it means.